rest-api/api_util.py

```diff
@@ -14,9 +14,10 @@
 # Role constants
 PTC = "ptc"
 HEALTHPRO = "healthpro"
+STOREFRONT = "storefront"
```

Comment:
Note that you'll need to add stuff to config_<env>.json in order to whitelist the appropriate service account for storefront in each environment.